admiral.app.js

```diff
@@ -130,10 +130,13 @@ function _initializeConfig(bag, next) {
   else
     bag.config.runMode = 'production';
 
-  if (bag.env.IGNORE_TLS_ERRORS && bag.env.IGNORE_TLS_ERRORS === 'true')
+  if (bag.env.IGNORE_TLS_ERRORS && bag.env.IGNORE_TLS_ERRORS === 'true') {
+    // disable TLS check for admiral itself
```

Comment:
why are we doing this?

Reply:
@ric03uec : admiral service was left out so disabled TLS there also consistency if the IGNORE_TLS_ERRORS flag is set

Reply:
got it